pkg/rhcos/vmware.go

```diff
@@ -0,0 +1,39 @@
+package rhcos
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/pkg/errors"
+)
+
+// VMWARE fetches the URL of the Red Hat Enterprise Linux CoreOS release.
+func VMWARE(ctx context.Context) (string, error) {
```

Comment:
@jstuever I think `VMWARE` should be `VMware` similar to `OpenStack`.

Reply:
The other providers vary but it looks like its based on "disk" name.  So maybe this makes more sense if it was `OVA` or `VMDK`?

Reply:
Looks like they all match the string in rhcos.json, case aside. Going with `VMware`.